### PR TITLE
Implement embeddable dashboard on Office 365

### PR DESCRIPTION
**Pull request by JuanGarriuz**
## Description

Implement the embeddable dashboard on Office 365-> dashboard tab and deprecate any use of kibana-integrations components
 
## Issues Resolved

[[Remove discover] Implement embeddable dashboard on Office 365 module](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6524)

# Test cases

> [!NOTE]
> The `global`, `dashboard`, and `discover` tests cases are the same as the mitre module: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6482

# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check rule.groups filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between inventory and dashboard tab

<details>

</details>

-  Check the filters persist between the inventory and dashboard tab

<details>

</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>

</details>

- Check rule.groups filter on tabs

<details>

</details>

- Check pinned agent filter on tabs

<details>

</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47

</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>

# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>

</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>

</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>

</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>

</details>

# Panel tab

> [!NOTE]
> The framework tab must work like latest stable version

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use the dropdown filters and check the filters applied and data showed | :black_circle: | :black_circle: | :black_circle: |
| Click the "Advanced filters" switch and use the search bar | :black_circle: | :black_circle: | :black_circle: |
| Use the search bar bottom filter bar | :black_circle: | :black_circle: | :black_circle: |
| Use the panels order row feature | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals  | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Use the inner panel data grid | :black_circle: | :black_circle: | :black_circle: |

-  Use the dropdown filters and check the filters applied and data showed
<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/62295a21-e218-4566-bf1b-a914ae852e7b

</details>

- Click the "Advanced filters" switch and use the search bar

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/e2a4bde6-92fc-4205-ba32-ebebc2ea6e6c

</details>

- Use the search bar bottom filter bar

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/a3902e67-73a7-4e5f-8a03-0fea5e17352b

</details>

- Use the panels order row feature

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/65c2a7a0-1998-4ec3-9241-62099b7c785b

</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered 

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/fdd20008-44ab-46bc-9e07-be77e3ef006b

</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals 

<details>

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/d9af64df-62e7-46ba-a3b1-3ae974c0f391

</details>

- Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/39ec0b39-969f-4dd5-b344-9ea3f313888a

</details>

- Use the inner panel data grid

<details>

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/8fcf7b6a-363c-4986-84d4-5ee80a4a9a95

</details>

# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>

</details>

- Apply filters using the search bar `add filter` feature

<details>

</details>

- Apply filters using the date picker

<details>

</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>

</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>

</details>

## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>

</details>

- Use chart options (three points: maximize, minimize, inspect) 

<details>

</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout (list and JSON views) | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>

</details>

- Check table pagination and rows per page

<details>

</details>

- Check document detail flyout (list and JSON views)

<details>

</details>

## Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by JuanGarriuz** [comment]
## Update 22/03 Drilldowns

### Ip 
#### Old

image


#### New

image


### Operation
#### Old

image


#### New

image

**Comment by yenienserrano** [comment]
# Test cases

> [!NOTE]
> The `global`, `dashboard`, and `discover` tests cases are the same as the mitre module: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6482

# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check rule.groups filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between inventory and dashboard tab

<details>


</details>

-  Check the filters persist between the inventory and dashboard tab

<details>


</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>


</details>

- Check rule.groups filter on tabs

<details>


</details>

- Check pinned agent filter on tabs

<details>


</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47


</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>


# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>


</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>


</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>


</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>


</details>

# Panel tab

> [!NOTE]
> The framework tab must work like latest stable version

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use the dropdown filters and check the filters applied and data showed | :black_circle: | :black_circle: | :black_circle: |
| Click the "Advanced filters" switch and use the search bar | :black_circle: | :black_circle: | :black_circle: |
| Use the search bar bottom filter bar | :black_circle: | :black_circle: | :black_circle: |
| Use the panels order row feature | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals  | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Use the inner panel data grid | :black_circle: | :black_circle: | :black_circle: |



-  Use the dropdown filters and check the filters applied and data showed
<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/62295a21-e218-4566-bf1b-a914ae852e7b



</details>

- Click the "Advanced filters" switch and use the search bar

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/e2a4bde6-92fc-4205-ba32-ebebc2ea6e6c

</details>


- Use the search bar bottom filter bar

<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/a3902e67-73a7-4e5f-8a03-0fea5e17352b


</details>

- Use the panels order row feature

<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/65c2a7a0-1998-4ec3-9241-62099b7c785b


</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered 

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/fdd20008-44ab-46bc-9e07-be77e3ef006b



</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals 

<details>



https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/d9af64df-62e7-46ba-a3b1-3ae974c0f391



</details>


- Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/39ec0b39-969f-4dd5-b344-9ea3f313888a

</details>

- Use the inner panel data grid

<details>


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/8fcf7b6a-363c-4986-84d4-5ee80a4a9a95


</details>



# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>


</details>

- Apply filters using the search bar `add filter` feature

<details>


</details>

- Apply filters using the date picker

<details>


</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>


</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>


</details>


## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>


</details>


- Use chart options (three points: maximize, minimize, inspect) 

<details>


</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout (list and JSON views) | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>


</details>

- Check table pagination and rows per page

<details>


</details>


- Check document detail flyout (list and JSON views)

<details>


</details>

**Comment by jbiset** [comment]
## CR: 

## Test UI:

> [!NOTE]
> The `global`, `dashboard`, and `discover` tests cases are the same as the mitre module: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6482

# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check rule.groups filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between inventory and dashboard tab

<details>


</details>

-  Check the filters persist between the inventory and dashboard tab

<details>


</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>


</details>

- Check rule.groups filter on tabs

<details>


</details>

- Check pinned agent filter on tabs

<details>


</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47


</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>


# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>


</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>


</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>


</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>


</details>

# Panel tab

> [!NOTE]
> The framework tab must work like latest stable version

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use the dropdown filters and check the filters applied and data showed | :black_circle: | :black_circle: | :black_circle: |
| Click the "Advanced filters" switch and use the search bar | :black_circle: | :black_circle: | :black_circle: |
| Use the search bar bottom filter bar | :black_circle: | :black_circle: | :black_circle: |
| Use the panels order row feature | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered | :black_circle: | :black_circle: | :black_circle: |
| Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals  | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed | :black_circle: | :black_circle: | :black_circle: |
| Use the inner panel data grid | :black_circle: | :black_circle: | :black_circle: |



-  Use the dropdown filters and check the filters applied and data showed
<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/62295a21-e218-4566-bf1b-a914ae852e7b



</details>

- Click the "Advanced filters" switch and use the search bar

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/e2a4bde6-92fc-4205-ba32-ebebc2ea6e6c

</details>


- Use the search bar bottom filter bar

<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/a3902e67-73a7-4e5f-8a03-0fea5e17352b


</details>

- Use the panels order row feature

<details> 


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/65c2a7a0-1998-4ec3-9241-62099b7c785b


</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and check the detail rendered 

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/fdd20008-44ab-46bc-9e07-be77e3ef006b



</details>

- Click on the panels row (Actors, Organizations, Repositories, Actions) and the totals 

<details>



https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/d9af64df-62e7-46ba-a3b1-3ae974c0f391



</details>


- Navigate on the panels row (Actors, Organizations, Repositories, Actions) and returns and check if the filter is applied/removed

<details> 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/39ec0b39-969f-4dd5-b344-9ea3f313888a

</details>

- Use the inner panel data grid

<details>


https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/8fcf7b6a-363c-4986-84d4-5ee80a4a9a95


</details>



# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>


</details>

- Apply filters using the search bar `add filter` feature

<details>


</details>

- Apply filters using the date picker

<details>


</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>


</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>


</details>


## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>


</details>


- Use chart options (three points: maximize, minimize, inspect) 

<details>


</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout (list and JSON views) | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>


</details>

- Check table pagination and rows per page

<details>


</details>


- Check document detail flyout (list and JSON views)

<details>


</details>

File: plugins/main/public/components/overview/office/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          ) : null}
+          {dataSource && results?.hits?.total > 0 ? (
+            <div className='office-365-dashboard-responsive'>
+              <SampleDataWarning />
+              <DashboardByRenderer
+                input={{
+                  viewMode: ViewMode.VIEW,
+                  panels: getKPIsPanel(dataSource.id),
+                  isFullScreenMode: false,
+                  filters: searchBarProps.filters ?? [],
```
Comment (jbiset) [inline plugins/main/public/components/overview/office/dashboard/dashboard.tsx:112]: Shouldn't the filters attribute receive the `fetchFilters`?

Comment (Machi3mfl) [reply plugins/main/public/components/overview/office/dashboard/dashboard.tsx]: Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6528/commits/5f60571ef95a2fb4f255c1301d36c59cfb270df7

File: plugins/main/public/components/overview/github/panel/config/drilldown-action.tsx [hunk 2]
```diff
@@ -176,15 +176,14 @@ export const DrilldownConfigAction = (drilldownProps: ModuleConfigProps) => {
...
-                      }
+                      onClick={e => {
+                        getCore().application.navigateToApp(rules.id, {
+                          path: `#/manager/?tab=rules&redirectRule=${value}`,
+                        });
+                      }}
                     >
                       {value}
-                    </EuiLink>
+                    </EuiLink >
```
Comment (yenienserrano) [inline plugins/main/public/components/overview/github/panel/config/drilldown-action.tsx:186]: Maybe these redirections can be done with the component that opensearch offers to show in the bottom corner to which link the link redirects to.

```
<RedirectAppLinks application={getCore().application}>
 <EuiLink
   href={`${rules.id}#/manager/?tab=rules&redirectRule=${value}`}
 >
    {value}
  </EuiLink>
</RedirectAppLinks>

```

Comment (Machi3mfl) [reply plugins/main/public/components/overview/github/panel/config/drilldown-action.tsx]: Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6528/commits/bd9097d22abcd6af60ad48db3530eaa252f080eb

**Review by yenienserrano — APPROVED** [review]
CR: 🟢 
Test: 🟢

**Review by jbiset — APPROVED** [review]

**Pull request merged**